hypershift-operator/main.go

```diff
@@ -228,6 +230,21 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 	}
 	log.Info("Using metrics set", "set", metricsSet.String())
 
+	// Convert RegistryOverrides flag to a map of string-to-slice of strings
+	registryOverrides := util.ConvertRegistryOverrideFlagDataType(opts.RegistryOverrides)
+
+	// Populate RegistryOverrides with ICSP from management cluster if no value is passed in the RegistryOverrides flag
+	if len(registryOverrides) == 0 {
+		client, err := cmdutil.GetClient()
```

Comment:
Is there a better way to get the MC client here or is it okay to use the command util for this?

Reply:
@sjenning - just FYI, using the `apiReadingClient` works so I removed the command util.